Makefile

```diff
@@ -2,20 +2,23 @@ PKGS=$(shell go list ./... | grep -v /vendor)
 SHELL_IMAGE=golang:1.8.3
 GIT_SHA=$(shell git rev-parse --verify HEAD)
 VERSION=$(shell cat VERSION)
+PWD=$(shell pwd)
 
 default: bindata compile
+
 compile:
-	@go build -o ${GOPATH}/bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
+	go build -o bin/kubicorn -ldflags "-X github.com/kris-nova/kubicorn/cmd.GitSha=${GIT_SHA} -X github.com/kris-nova/kubicorn/cmd.Version=${VERSION}" main.go
```

Comment:
@kris-nova Would it be acceptable here to add a `make install` step which places the built files in `${GOPATH}/bin/kubicorn`? I agree that `make` should be equivalent to a `make build`. A `make all` step could also be added to replace the current behavior of plain `make` if you think `make build install` would be annoying.